### PR TITLE
add workload identity

### PR DESCRIPTION
**Pull request by sunkickr**
## Description

add workload identity to inspect command metadata

## 🎟 Issue(s)

- https://github.com/astronomer/astro-cli/issues/1242

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

## 📋 Checklist

- [ ] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `make test` before taking out of draft
- [ ] Ran `make lint` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: cloud/deployment/inspect/inspect.go [hunk 1]
```diff
@@ -372,3 +375,24 @@ func getTemplate(formattedDeployment *FormattedDeployment) FormattedDeployment {
 
 	return template
 }
+
+func getWorkloadIdentity(de *astro.Deployment) string {
+	// deployment workload identity only applies to AWS and GCP for now
+	fmt.Println(de.Cluster.CloudProvider)
```
Comment (kushalmalani) [inline cloud/deployment/inspect/inspect.go:381]: Do we need this print statement? or this was just for debugging?

File: cloud/deployment/inspect/inspect.go [hunk 2]
```diff
@@ -84,6 +85,7 @@ var (
 const (
 	jsonFormat    = "json"
 	notApplicable = "N/A"
+	gcpMax        = 30
```
Comment (kushalmalani) [inline cloud/deployment/inspect/inspect.go:88]: Can you add a comment of what this field is used for? Maybe rename it also?

**Review by kushalmalani — APPROVED** [review]
Left a couple of nits

File: cloud/deployment/inspect/inspect.go (hunk 1 above)
Comment (sunkickr) [reply cloud/deployment/inspect/inspect.go:381]: only for debugging

**Pull request merged**